on_boot.d/99-lego-install.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+UDM_LEGO_PATH=/mnt/data/ssl
```

Comment:
Maybe you should change this or change the README

Reply:
Do you happen to know the difference between `/mnt/data` and `/mnt/data_ext` in terms of durability on the UDMP? I wasn't sure if there are scenarios where one is more likely to stick around than the other. I'm open to moving stuff to a more standard location.

Reply:
Not sure about it, but I think there is no difference. I just used it because was created (at least after install on_boot package) and makes sense for me have it there.